src/tcgwars/logic/impl/gen4/LegendsAwakened.groovy

```diff
@@ -637,8 +640,10 @@ public enum LegendsAwakened implements LogicCardInfo {
             onAttack {
               damage 60
               def maxCount = Math.min(self.cards.filterByEnergyType(M).size(), opp.bench.size())
-              multiSelect(opp.bench, maxCount, text).each {
-                damage 20, it
+              if (maxCount != 0) {
```

Comment:
This was actually included because the attack text doesn't include "you may." If there is at least 1 [M] on probopass and your opponent has a bench, you would have to select at least 1 benched pokemon. (No ruling on probopass explicitly, but is consistent with rulings on team galactic's wager, regigigas lv x, and looker's investigation, among others)

Reply:
Which is why you set the `min` for `multiSelect` to 1, but per the source code of the method it calls to actually do the work, the code if the value passed to `selectMultiPokemon`'s `max` is smaller than the `min` it ensures you select that smaller amount, and if the `max` is 0 it just skips the selection loop entirely. This `if` condition doesn't do anything that code doesn't already do.

Reply:
got it, in the process of fixing.